### ISSUE TITLE
Switch to line in not working 

### ISSUE THREAD

**Issue by eduhh94** [SoCo/SoCo#661]
Hello dear SoCo team,

I wanted to use your API to switch to line-in but it's not working. Any idea why?
Volume worked fine.

```
my_zone.switch_to_line_in()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/soco/core.py", line 1079, in switch_to_line_in
    ('CurrentURIMetaData', '')
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/soco/services.py", line 213, in _dispatcher
    return self.send_command(action, *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/soco/services.py", line 498, in send_command
    self.handle_upnp_error(response.text)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/soco/services.py", line 560, in handle_upnp_error
    error_xml=xml_error
soco.exceptions.SoCoUPnPException: UPnP Error 714 received: Illegal MIME-Type from 192.168.178.45
```

Discovery is also not working and sonos.player_name shows nothing.
```

>>> from soco import SoCo
>>> sonos = SoCo('192.168.178.45')
>>> sonos.player_name
>>> 
```

**Comment by rleeden** [comment]
Your sonos probably updated to version 10.1
See [Issue 660](https://github.com/SoCo/SoCo/issues/660) for discussion on this and a possible fix

**Comment by svrooij** [comment]
this seems like an easy fix.

Just call [setavtransporturi](https://svrooij.io/sonos-api-docs/services/av-transport.html#setavtransporturi) with:

- InstanceID = `0`
- CurrentURI = `x-rincon-stream:{uuid}` [see this to load the uuid](https://github.com/svrooij/node-sonos-ts/blob/master/src/sonos-device.ts#L276-L282)
- CurrentURIMetaData = ``

Can anyone point me to where all the requests are defined, then I’ll create a PR to fix this

**Comment by pwt** [comment]
This issue is no longer present, so I'm going to close it. It was a consequence of #660, which was resolved in PR #662.
